errors/B0004.md

```diff
@@ -103,18 +102,15 @@ fn main() {
 ```
 
 A similar problem occurs when the [`GlobalTransform`] component is missing.
-However, when a parent [`GlobalTransform`] is missing,
-it will simply prevent all transform propagation,
-including when updating the [`Transform`] component of the child.
+However, when it will be automatically inserted whenever `Transform` is
```

Comment:
```suggestion
However, it will be automatically inserted whenever `Transform` is
```
I think we should also keep the lines you removed above this. It's less likely to be an issue now but it's still true.